MapboxNavigation/JunctionView.swift

```diff
@@ -49,19 +54,31 @@ public class JunctionView: UIImageView {
                 guard let guidanceViewImageURL = URL(string: stringURL) else { return }
                 imageRepository.imageWithURL(guidanceViewImageURL, cacheKey: guidanceView.cacheKey!) { [unowned self] (downloadedImage) in
                     
-                    self.isCurrentlyVisible = true
-                    self.isHidden = !self.isCurrentlyVisible
```

Comment:
Does this affect whether the view is visible while the image is still loading? If the view is currently visible, the view always gets hidden here, so what happens if there are two junctions in short succession that both need the junction view to be visible?

Reply:
This code is from Fabian's original PR, I just moved it into a dispatch block since the function can be invoked off the main thread. As to the substance of your comment I think you are right that the original code was to prevent the view from being visible when the image wasn't loaded yet. However, I don't think we need to deal with that in the loading code and let the show/hide logic sort it out. It works fine in my testing but I haven't been able to come up with a test case that includes having two maneuvers show up in quick succession.